politeiawww/api/www/v1/v1.go

```diff
@@ -1360,3 +1369,25 @@ type WSSubscribe struct {
 type WSPing struct {
 	Timestamp int64 `json:"timestamp"` // Server side timestamp
 }
+
```

Comment:
Where's the api documentation?  Also, can we get some more context in these comments.  It's not clear from the comments how these routes work.